src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4330,25 +4330,25 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>lambda expression</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion1" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 1.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 1. Please use language version {1} or greater.</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion2" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 2.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 2. Please use language version {1} or greater.</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion3" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 3.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 3. Please use language version {1} or greater.</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion4" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 4.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 4. Please use language version {1} or greater.</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion5" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 5.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 5. Please use language version {1} or greater.</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion6" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 6.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 6. Please use language version {1} or greater.</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion7" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 7.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 7. Please use language version {1} or greater.</value>
```

Comment:
This is a follow-up on the last PR. `ERR_FeatureNotAvailableInVersion7_1` was added and it had this extra space. Then Omar pointed out all the previous versions had the same typo.